pom.xml

```diff
@@ -9,17 +9,17 @@
         <jacoco.version>0.8.7</jacoco.version>
         <gpg.skip>true</gpg.skip>
         <!--
-        M4 does not fail tests if beforeAll fails
-        M5 fails in integration tests with a strange error
-        -->
+            M4 does not fail tests if beforeAll fails
+            M5 fails in integration tests with a strange error
+            -->
         <surefire.and.failsafe.plugin.version>3.0.0-M3</surefire.and.failsafe.plugin.version>
-        <vs-commom-document-files.version>2.2.0</vs-commom-document-files.version>
+        <vs-commom-document-files.version>4.0.0-SNAPSHOT</vs-commom-document-files.version>
```

Comment:
Do not forget to update version once you release vs-common-document-files.

Reply:
:heavy_check_mark: